miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -1781,6 +1781,7 @@ public static Pool to(PoolDto dto) {
       PoolableElementView view = new PoolableElementView();
       view.setDilutionId(dilution.getId());
       view.setDilutionName(dilution.getName());
+      view.setDilutionVolumeUsed(dilution.getVolumeUsed() == null ? null : Double.valueOf(dilution.getVolumeUsed()));
```

Comment:
`dilution#getVolumeUsed` already returns a double, so this could just be `view.setDilutionVolumeUsed(dilution.getVolumeUsed())`

Reply:
`dilution` is a `DilutionDto` here, which stores `volumeUsed` as a String.

Reply:
Oh, right. :+1: